PathOfBuildingCommunity/PathOfBuilding
# Implemented Stationary Keystones/Notables

Fixes issues: #269, #263

To Test:

For #269 - create
```
Nature Patience Simulator
Cobalt Jewel
LevelReq: 1
Implicits: 0
Gain 2 Grasping Vines each second while stationary
2% chance to deal Double Damage per Grasping Vine
1% less damage taken per Grasping Vine
```

1. Put in an empty jewel socket
2. Check that Configuration tab now has "Are you stationary?" option
3. Pick a Skill Gem for DPS (e.g. Cyclone) and Equip a Staff (e.g. Agnerod East unique) - note the Combined DPS
4. Increase Configuration Stationary Counter to 4 (see Combined DPS increase)
5. In the Calcs tab, Hover over MH Hit Damage for Physical and see that it has a Double Damage chance equal to 2 times the Stationary Counter (set to 4) * 2% chance to deal Double Damage = 16% (1.16)
6. In the Calcs tab, hover over Physical Hit/DoT numbers in Damage Taken section and see that there is a section for 8% less damage taken with a Note of (1% per Grasping Vines Count). Since we get 2 vines per second and set Stationary Counter to 4 we get 8% less damage.
7. Verify that going above Stationary Count value of 5 is meaningless (capped at 5)
8. Remove item from jewel socket (verify Configuration tab no longer has Stationary option)

For #263 - create
```
Expansive Might Simulator
Cobalt Jewel
LevelReq: 1
Implicits: 0
While stationary, gain 10% increased Area of Effect every second, up to a maximum of 50%
20% increased damage with maces, scepters, or staves
```

1. Pick a Skill Gem for DPS (e.g. Cyclone) and Equip a Staff (e.g. Agnerod East unique) - note the Combined DPS
2. Put jewel in an empty jewel socket (note DPS increased - we are using a staff and jewel gives 20% increased damage with maces, scepters, or staves)
3. Check that Configuration tab now has "Are you stationary?" option
4. Increase Configuration Stationary Counter to 4 (Combined DPS should not increase, but AoE Radius should)
5. In the Calcs tab, hover of "Area of Effect Mod" value and verify the 40% increase exists from jewel. (10% per stationary second which we set to 4).
6. Verify that going above Stationary Count value of 5 is meaningless (capped at 5)
